Services/AccessControl/interfaces/interface.ilRBACAccessHandler.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+/**
+ * Class ilAccessHandler
+ *
+ * Checks access for ILIAS objects
+ *
+ * @author  Alex Killing <alex.killing@gmx.de>
+ * @author  Sascha Hofmann <saschahofmann@gmx.de>
+ * @version $Id$
+ *
+ * @ingroup ServicesAccessControl
+ */
+interface ilRBACAccessHandler {
```

Comment:
Thanks A LOT.

Seems to me, as if the caching stuff should be removed in the long run, so consumers cannot tamper with that.

This also should be a good starting point to work on the mechanics and encoding of the RBAC on the code-side in general.